backend/app.js

```diff
@@ -11,19 +11,31 @@ const urlencodedParser = bodyParser.urlencoded({ extended: false })
 const app = express()
 const port = 3000
 
-require('./bot');
-
 const addCrossDomainHeaders = function(req, res, next) {
-    res.header('Access-Control-Allow-Origin', "*")
+    res.header('Access-Control-Allow-Origin', '*')
     res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE')
     res.header('Access-Control-Allow-Headers', 'Content-Type')
-    next();
+    next()
 }
 
 app.use(addCrossDomainHeaders)
 app.use(urlencodedParser)
 app.use(jsonParser)
 
+
+const botFactory = require('./bot')
+let bot
+botFactory.makeBot(process.env.FOOSBOT_TOKEN, process.env.FOOS_CHANNEL_NAME)
+    .then(botInstance => {
+        bot = botInstance
```

Comment:
not sure if this is a JS antipattern, but dunno how to make it cleaner. I had to handle the makeBot creation promise, otherwise i'd end up with `(node:49762) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:49762) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
` in case of missing env.

Reply:
No, I think this is fine.